gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/GobblinMCEWriter.java

```diff
@@ -331,7 +336,7 @@ void writeWithMetadataWriters(
         } catch (Exception e) {
           meetException = true;
           writer.reset(dbName, tableName);
-          addOrThrowException(e, tableString, dbName, tableName);
+          addOrThrowException(e, tableString, dbName, tableName, writer.getClass().getName());
```

Comment:
Since we disable the following writer as well, not sure whether you want to add the following writer to the exception as well, depends on you

Reply:
Not sure what you mean? It's adding the name of the writer class that failed so that we know if failure happened in iceberg writer or in hive writer.

Reply:
yeah,  basically I try to understand the meaning here: whether it's represent where the failure happens or it represents which writer do not successfully flush. If it's later, then we probably want to add following writers' name there as well? i.e. if hive writer fail, we reset iceberg writer, which basically iceberg writer also not flush as well

Reply:
Ah okay I understand, yeah I was more thinking of giving the name of the writer that caused the issue, but I think it makes sense to include all writers which did not write successfully, I will do that.

Reply:
Updated to include a list of writers that were failed/skipped.